src/Parser/HolyPriest/CPM_ABILITIES.js

```diff
@@ -15,7 +15,8 @@ const CPM_ABILITIES = [
     spell: SPELLS.PRAYER_OF_MENDING_CAST,
     name: 'Prayer of Mending',
     category: SPELL_CATEGORY.ROTATIONAL,
-    getCooldown: (haste, combatant) => (12 - (combatant.hasTalent(SPELLS.PIETY_TALENT.id) ? 2 : 0)) / (1 + haste),
+    getCooldown: (haste, combatant) => ((12 - (combatant.hasTalent(SPELLS.PIETY_TALENT.id) ? 2 : 0)) + 1.5) / (1 + haste), // +1.5 for base cast time
+    recommendedCastEfficiency: 0.70, // There's a lot of things that can potentially delay PoM: movement, heavy damage, etc. It is important to maintain high uptime but delaying a few GCDs is not bad.
```

Comment:
Isn't this a bit double though? You're already nerfing the max CPM by including the cast time, if casting it is actually important but missing a few GCDs isn't a big issue than the default cast efficiency should be enough right? It's aimed to allow exactly that.